Example/Source/Examples/BaseBrickController.swift

```diff
@@ -15,21 +15,21 @@ class BaseBrickController: BrickViewController {
 
     var isBehaviorEnabled = true {
         didSet {
-            updateBehavior()
+            updateBehavior(false)
         }
     }
 
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
         self.view.backgroundColor = .brickBackground
-        updateBehavior()
+        updateBehavior(true)
```

Comment:
It's not clear to me reading this code what the boolean flag does at the call site. It might be clearer to say something like this so that others who read this code know when the flag should be true or not:
```swift
updateBehavior(isViewAppearing: true)
```
Conversely, it might also be simpler to just put the `self.brickCollectionView.invalidateBricks()` wherever you'd pass false, instead of putting it in `updateBehavior()`. That way the flag isn't even necessary.